src/scss/grommet-core/_objects.layer.scss

```diff
@@ -53,6 +53,7 @@
     top: 0px;
     right: 0px;
     z-index: 1;
+    padding: halve($inuit-base-spacing-unit);
```

Comment:
Im not sure why adding padding will fix the issue. Can we try and remove the absolute positioning of the closer icon and add that as part of a box with justify right?